pyomo/contrib/cspline_external/cspline_parameters.py

```diff
@@ -198,6 +231,18 @@ def f(self, x):
         s = self.segment(x)
         return self.a1[s] + self.a2[s] * x + self.a3[s] * x**2 + self.a4[s] * x**3
 
+    def fx(self, x):
+        """Get f(x)
+
+        Args:
+            x: location, numpy array float
+
+        Returns:
+            f(x) numpy array if x is numpy array or float
+        """
+        s = self.segment(x)
+        return self.a2[s] + 2 * self.a3[s] * x + 3 * self.a4[s] * x**2
+
```

Comment:
`def f` and `def fx` both "get f(x)" but f(x) is different in both scenarios. That strikes me as confusing.

Reply:
fx is df/dx, so dfdx is probably a better name, and now I see I forgot to edit the doc string when I copied it.